components/audioinput/client.go

```diff
@@ -42,6 +42,9 @@ func NewClientFromConn(
 	name resource.Name,
 	logger logging.Logger,
 ) (AudioInput, error) {
+	// TODO(RSDK-6340): This client might still try to create audio streams after this
+	// context is canceled. These subsequent audio streams will not work. To fix this,
+	// use a channel instead of a context like we do in `component/audioinput/client.go`
```

Comment:
assuming we decide to merge this PR, we should make the same change for audio input since it has the same bug

Reply:
I added a link to this ticket and a comment in [our epic](https://viam.atlassian.net/browse/RSDK-2282) to add audio support. As of now, this isn't urgent since we don't advertise audio support.